packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/tests/index.test.js

```diff
@@ -362,4 +362,96 @@ describe('BlocksEditor toolbar', () => {
     // The dropdown should show only one option selected which is the block content in the first row
     expect(within(headingsDropdown).getByText(/heading 1/i)).toBeInTheDocument();
   });
+
+  it('slits the parent list when converting a list item to another type', async () => {
```

Comment:
typo
`it('slits the parent list when converting a list item to another type', async () => {`